server/src/main.rs

```diff
@@ -26,22 +40,28 @@ pub struct ArrowPricing {}
 #[tonic::async_trait]
 impl Pricing for ArrowPricing {
     /// Get pricing for a given query.
+    #[cfg(not(tarpaulin_include))]
     async fn get_pricing(
         &self,
-        request: tonic::Request<pricing_grpc::PricingRequest>,
+        request: tonic::Request<pricing_grpc::PricingRequests>,
     ) -> Result<tonic::Response<pricing_grpc::PricingResponse>, tonic::Status> {
-        grpc_info!("Received request: {:?}", request);
         let query = request.into_inner();
-        grpc_debug!("Getting pricing for query: {:?}", query);
-        let price = get_pricing(query);
-        grpc_debug!("Pricing computed: {}", price);
-        let response = pricing_grpc::PricingResponse { price };
-        grpc_debug!("Returning response: {:?}", response);
-        grpc_info!("Successfully computed pricing; returning response");
-        Ok(tonic::Response::new(response))
+        info!("Received pricing request");
```

Comment:
make this a debug call, and end it with punctuation

also nice to prepend (get_pricing) to the message so it's easily grep'd in logs

Reply:
[17d9bf3](https://github.com/Arrow-air/svc-pricing/pull/19/commits/17d9bf3b229051e65e10107c1e7d5756c5de34e0)